README.md

```diff
@@ -1,5 +1,131 @@
-# Minishift Fedora VM
+<a name="creating-a-minishift-fedora-iso"></a>
+# Minishift Fedora ISO
 
 This repository contains all the instructions and code to build a Live ISO based on Fedora
 which can be used by [minishift](https://github.com/minishift/minishift) as an alternative to
-the boot2docker and CentOS ISO.
+the boot2docker ISO.
+
+----
+
+<!-- MarkdownTOC -->
+
+- [Building the Fedora ISO](#building-the-fedora-iso)
+	- [On Fedora](#on-fedora)
+		- [Prerequisites](#prerequisites)
+		- [Building the ISO](#building-the-iso)
+	- [On hosts _other than Fedora_ \(OS X, Windows, CentOS ...\)](#non-fedora-hosts)
+		- [Prerequisites](#prerequisites-1)
+		- [Building the ISO](#building-the-iso-1)
+  - [Manual release](#manual-release)
+- [Further reading](#further-reading)
+- [Community](#community)
+
+<!-- /MarkdownTOC -->
+
+----
+
+<a name="building-the-fedora-iso"></a>
+## Building the Fedora ISO
+
+The following contains instructions on how to build the default (Fedora based) ISO.
```

Comment:
`the default (Fedora based) ISO` is confusing me. Fedora ISO isn't our default ISO.

 Probably line should be
```
The following contains instructions on how to build the Fedora based ISO.
```